easybuild/easyconfigs/c/cppGSL/cppGSL-4.0.0-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'CMakeMake'
+name = 'cppGSL'
+version = '4.0.0'
+
+description = """
+The Guidelines Support Library (GSL) contains functions and types that are suggested for use by the C++ Core Guidelines maintained by the Standard C++ Foundation. This repo contains Microsoft's implementation of GSL.
+
+The entire implementation is provided inline in the headers under the gsl directory. The implementation generally assumes a platform that implements C++14 support.
+
+While some types have been broken out into their own headers (e.g. gsl/span), it is simplest to just include gsl/gsl and gain access to the entire library.
+"""
+homepage = 'https://github.com/microsoft/GSL'
+source_urls = ['https://github.com/microsoft/GSL/archive']
+sources = ['v%(version)s.zip']
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+test_cmd = 'ctest'
+runtest = '-C Debug'
```

Comment:
Do we really want to specify the debug config specifically here? I would have thought that the release would be appropriate since that's what one would be using in production. 
Which I would suspect would run fine with just;

```
runtest = 'tests'
```

as such a make target is typically available.